src/pmacfiltercontrol/pmacFilterControlWrapper.py

```diff
@@ -220,20 +220,22 @@ def __init__(
             on_update=self._set_histogram_scale,
         )
 
-        self.autosave_exists: bool = self._check_autosave_file_exists()
+        self.autosave_exists: bool = self._check_autosave_file_exists(
+            self.autosave_pos_file_path
+        )
+
         self._autosave_pos_dict: Dict[str, float] = {}
 
         if self.autosave_exists:
```

Comment:
You could create a `Path` object on init to simplify a lot of this code, e.g. (I can't add the creation of the Path where it should be because that line hasn't changed)
```suggestion
        self.autosave_file = Path(autosave_file_path)
...
        if self.autosave_file.exists()
```

I also think it doesn't make sense to have `pos` in these variable names anymore as it has more than that.